vmdb/app/controllers/application_controller/current_user.rb

```diff
@@ -21,20 +21,27 @@ def current_user=(db_user)
       session[:userid]    = nil
       session[:username]  = nil
     end
+    self.current_group  = db_user.try(:current_group)
+    self.current_eligible_groups = db_user.try(:eligible_groups)
```

Comment:
DashboardController was using `db_user.miq_groups` to compute the eligible groups.  Is `eligible_groups` the correct method to try?